### DISCUSSION TITLE
Can no longer use embedded many-to-many relations in combination with Postgres?

### DISCUSSION THREAD

**Discussion by alucardu** [prisma/prisma#13042] [Q&A]
I wanted to jump back in my old project but my editor is now showing me an error on my schema and Prisma won't run the `prisma generate` action.

> Error validating: Embedded many-to-many relations are not supported on Postgres. Please use the syntax defined in https://pris.ly/d/relational-database-many-to-many

```
model Movie {
  id             Int            @id @default(autoincrement())
  movieRating    MovieRating[]
}

model MovieRating { 
  id           Int              @id @default(autoincrement())
  movieId      Int?
  movie        Movie[]          @relation(fields: [movieId], references: [id])
}
```

This once worked since I had my app in production. (I removed some lines from the code block, here's the whole schema > https://github.com/alucardu/movieseat-react/blob/master/prisma/schema.prisma).

The documentation doesn't help because it doesn't show an example of a embedded many-to-many relation.

**Comment by janpio** [comment]
There are no embedded many to many relations for relational databases, only MongoDB. Something weird is going on here.

What kind of relation do you _want_ to exist between Movie and MoveRating? One movie can have many ratings I assume?

**Reply by janpio** [reply]
Based on that assumption, this is probably what your schema _should_ look like:

```
model Movie {
  id          Int           @id @default(autoincrement())
  movieRating MovieRating[]
}

model MovieRating {
  id      Int   @id @default(autoincrement())
  movieId Int
  movie   Movie @relation(fields: [movieId], references: [id])
}
```

Is there a reason why `movieId` was marked optional?
Does the current relation code work with the old Prisma version?

**Reply by alucardu** [reply]
I'm not sure why it `movieId`was marked optional, it shouldn't have been. Also not sure why `movie` was an table, a `movieRating` has 1 movie. Kinda strange.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
